Sources/Document/Json/JSONCounter.swift

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+public protocol YorkieCountable: BinaryInteger {
+    var littleEndian: Self { get }
+    func addingReportingOverflow(_ other: Self) -> (partialValue: Self, overflow: Bool)
+}
+
+extension Int32: YorkieCountable {}
+extension Int64: YorkieCountable {}
```

Comment:
YorkieCountable is used for both CRDT layers and JSON layers, so moving it to another appropriate position would be good.

Reply:
I made another file named "YorkieCountable.swift"